creator-node/src/routes/files.js

```diff
@@ -246,6 +247,81 @@ const getCID = async (req, res) => {
     // continue
   }
 
+  /**
+   * If not found on FS at storagePath, check legacyStoragePath
+   */
+  if (!fileFoundOnFS) {
```

Comment:
these if/else conditions look duplicated from 205-247. can we do combine it all into something like this instead?
```
try {
   let fsStats = await fs.stat(storagePath)
   if (!fsStats) {
       const legacyStoragePath = DiskManager.computeLegacyFilePath(CID)
       fsStats = await fs.stat(legacyStoragePath)
       storagePath = legacyStoragePath
       // also handle case if error happens when computing legacy path 
   }

   // etc rest of code
} catch (e) {
  // ...
}
```

Reply:
yeah i can DRY this up
originally was planning on shipping this asap but have a short term fix instead so can go back and improve now
good call